tests/queries/0_stateless/01271_show_privileges.reference

```diff
@@ -138,6 +138,7 @@ SYSTEM THREAD FUZZER	['SYSTEM START THREAD FUZZER','SYSTEM STOP THREAD FUZZER','
 SYSTEM UNFREEZE	['SYSTEM UNFREEZE']	GLOBAL	SYSTEM
 SYSTEM	[]	\N	ALL
 dictGet	['dictHas','dictGetHierarchy','dictIsIn']	DICTIONARY	ALL
+displaySecretsInShowAndSelect	[]	GLOBAL	ALL
```

Comment:
Shouldn't it be `DISPLAY_SECRETS_IN_SHOW_AND_SELECT` like other grants? Since seems that `camelCase` is used only for functions.